beam/task/types/src/Rhyolite/Task/Beam.hs

```diff
@@ -0,0 +1,39 @@
+{-# Language DeriveAnyClass #-}
+{-# Language DeriveGeneric #-}
+{-# Language FlexibleInstances #-}
+{-# Language FunctionalDependencies #-}
+{-# Language KindSignatures #-}
+{-# Language RankNTypes #-}
+{-# Language StandaloneDeriving #-}
+{-# Language TemplateHaskell #-}
+{-|
+Description : Contains definitions for tasks, and tables that have tasks.
```

Comment:
What is "and tables that have tasks" referring to? I only see the `Task` type --- is GitHub not showing me the full diff? 🥴